### DISCUSSION TITLE
the question  about the calculation of SNR in test_py3gpp file

### DISCUSSION THREAD

**Discussion by Giga7756** [catkira/py3gpp#65] [General]
Hi：
    The index of SNR is caculated  in test_py3gpp file. The calculation formula is：
    SNR = 10*np.log10((np.linalg.norm(waveform) / np.linalg.norm(noise)))
   
   I think the formula should be:
    SNR = **20***np.log10((np.linalg.norm(waveform) / np.linalg.norm(noise)))

   what do you think？thanks

**Comment by catkira** [comment]
Hi,
yes I think You are correct. Do You want to make a PR to fix it?

**Reply by Giga7756** [reply]
Hi,
   Thanks very mauch for your reply. I am a communication system engineer who has been working for 7 years. But I am a beginner of the Github. To avoid errors in the github，you can do it？
   I am very happy to discuss the issue with you. And hope to have more opportunities to discuss with you
